TIKEBook_Lesson2.ipynb

```diff
@@ -0,0 +1,593 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    plt.xlim(1347,1350) #zoom in
</pre><p> </p><p>Why? I mean, yes, this section does look like a transit, but what motivates our decision to only look between 1347 and 1350?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>well without zooming in the curve looks very uninteresting, but i will remove it. changed the example target anyway so this zoom doesn't apply. </p>